spyder_notebook/notebookplugin.py

```diff
@@ -48,6 +48,7 @@ class NotebookPlugin(SpyderPluginWidget):
     """IPython Notebook plugin."""
 
     CONF_SECTION = 'notebook'
+    CONF_DEFAULTS = {'recent_notebooks': []}

```

Comment:
Did you check if there aren't more things we need to add here? Sorry, I don't remember if we try to save more config options in this plugin.

Reply:
That is a good question. I checked and as far as I can see, that is the only config option inside the plugin.

Reply:
Ok, thanks for checking that @jitseniesen!

Please merge and release 0.2.1 with this one.